CHANGELOG.md

```diff
@@ -1,3 +1,23 @@
+# dbt_twitter_source v0.10.1
+
```

Comment:
Should I create a schema change matrix for adding `segment_name` to the tmp models?

Reply:
Good question--I lean towards no since we don't track the tmp models in the dbt docs, but maybe let's bring it up at standup.